src/components/table/table.tsx

```diff
@@ -172,6 +173,19 @@ export class Table {
         this.updateMaxPage();
     }
 
+    @Watch('page')
+    public pageChanged() {
+        if (!this.tabulator) {
```

Comment:
Just a small question. When will `this.tabulator` becomes undefined?

Reply:
I'm not sure, but since it is tested in the other watch methods or `this.tabulator?` was used, I thought it was best to test it 😛

Reply:
I don't think it ever _becomes_ undefined, but it starts out as undefined, and is only defined after the component has loaded and rendered, in `initTabulatorComponent` on line 268. The `page` watcher will almost certainly have fired at least once by then, as I think these watchers always fire when the component loads. Without this check, line 182 below would cause an error.

Reply:
Ok, thanks for the explanation.  In the component lifecycle methods I know that the watcher is fired before the render method. For some reason without this check, line 182 didn't cause an error. The code in the watcher loads when `page` changes right?.

Reply:
Hmm… then I guess the watcher didn't fire when the component loaded. Interesting. Did you give the `page` property a value from the beginning? If not, that could be the reason, but it might also be that the watcher simply doesn't fire until the initial value is changed.

Regardless, having the check is a very small cost for avoiding potential errors, so I think we should keep it.

Reply:
Can you merge this PR then. I approved it. It works perfect as it should